### ISSUE TITLE
add examples to tests

### ISSUE THREAD

**Issue by scls19fr** [frictionlessdata/TableSchema.jl#39]
Running https://github.com/frictionlessdata/TableSchema.jl/tree/master/examples from unit tests could ensure that examples are running correctly

**Comment by roll** [comment]
@loleg 
Yes, it's just a usual for the libs folder structure but not a requirement
